client/src/DbtLanguageClientManager.ts

```diff
@@ -93,7 +94,10 @@ export class DbtLanguageClientManager {
       try {
         await workspace.fs.stat(currentUri.with({ path: `${currentUri.path}/${DBT_PROJECT_YML}` }));
         const oneLevelUpPath = Uri.joinPath(currentUri, '..').path;
-        if (!ExtensionClient.DEFAULT_PACKAGES_PATHS.some(p => oneLevelUpPath.endsWith(p))) {
+        if (
+          !ExtensionClient.DEFAULT_PACKAGES_PATHS.some(p => oneLevelUpPath.toLocaleLowerCase().endsWith(p)) &&
```

Comment:
Do we really need call `.toLocaleLowerCase()`? Is it Windows specific?
If dbt consider some folder like `Dbt_Packages` as valid?

Reply:
Unfortunately dbt consider folder like `Dbt_Packages` as valid. I think it is rare case and here we will just not run language server for such folders.